kodiswift/__init__.py

```diff
@@ -66,6 +66,11 @@ def func(*args, **kwargs):
     xbmcplugin = _Module(xbmcplugin)
     xbmcaddon = _Module(xbmcaddon)
     xbmcvfs = _Module(xbmcvfs)
+    sys.modules['xbmc'] = xbmc
+    sys.modules['xbmcgui'] = xbmcgui
+    sys.modules['xbmcplugin'] = xbmcplugin
+    sys.modules['xbmcaddon'] = xbmcaddon
+    sys.modules['xbmcvfs'] = xbmcvfs
```

Comment:
This is done to be able to fully mocks xbmc libraries even for non kodiswift's libraries